pkg/apis/migration/v1alpha1/directimagemigration_types.go

```diff
@@ -89,6 +92,20 @@ func (r *DirectImageMigration) GetDestinationCluster(client k8sclient.Client) (*
 	return GetCluster(client, r.Spec.DestMigClusterRef)
 }
 
+// Get the MigMigration that owns this DirectImageMigration. If not owned, return nil.
+func (r *DirectImageMigration) GetOwner(client k8sclient.Client) (*MigMigration, error) {
+	owner := &MigMigration{}
+	ownerRefs := r.GetOwnerReferences()
+	if len(ownerRefs) > 0 {
+		ownerRef := types.NamespacedName{Name: ownerRefs[0].Name, Namespace: r.Namespace}
+		err := client.Get(context.TODO(), ownerRef, owner)
+		if err != nil {
+			return nil, liberr.Wrap(err)
+		}
+	}
+	return owner, nil
```

Comment:
Make sure that any code that deals with the returned owner acts sanely if the owner is an empty `MigMigration{}`, since there will be no owner for standalone DIM resoruces.

Reply:
+1, will go make sure this is handled properly